ci/README.md

```diff
@@ -28,7 +28,7 @@ The `jenkins` target is excuted on every branch build by jenkins. It runs
   * `provision.killStaleTestProcesses()`
   * `provision.installMesos()`
   * `compileAndTest()`
-  * `createPackages()`
+  * `build()`
 
 The `phabricator` target runs the `jenkins` pipeline followed by Phabricator
```

Comment:
I know it is not connected to your changes but this part of readme is probably outdated because it mentions phabricator... ?